### PR TITLE
[DOC] Create single Using Operators section

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

**Documentation**
Consolidated the operators content into a single _Using Operators_ section in the _Using Guide_.
Operators introduced at the start of the guide. This chapter describes how they are used, configured.

The content is largely the same, with a few minor edits and additions.
12 files are now removed.

**Cluster Operator**
- assembly-operators-cluster-operator.adoc renamed assembly-using-the-cluster-operator.adoc + removed unnecessary assembly context
- dropped the intro (con-overview-components-cluster-operator.adoc) that is also provdied at the start of the guide
- removed con-cluster-operator-reconciliation.adoc and added content to ref-cluster-operator-config.adoc
- removed con-cluster-operator-rbac.adoc and added content to ref-cluster-operator-config.adoc

**Topic Operator**
- removed assembly-deploying-the-topic-operator.adoc (moving content into ref-topic-operator.adoc or removing if replicated)
-- removed replication of operator overview - introduced at start of guide
-- removed _Identifying a Kafka cluster for topic handling_ (con-topic-operator-cluster-label.adoc) and moved content into ref-topic-operator.adoc
-- removed _Understanding the Topic Operator_ (con-how-the-topic-operator-works.adoc) and moved content into ref-topic-operator.adoc (title changed to _Handling topic changes_)
- moved proc-topic-operator-with-resource-requests-limits.adoc

**User Operator**
- removed assembly-user-operator.adoc and unecessary assembly context
- dropped the intro (con-overview-components-user-operator.adoc) that is provdied at the start of the guide
- removed con-user-operator-cluster-label.adoc and added content to ref-topic-operator.adoc
- moved proc-user-operator-with-resource-requests-limits.adoc 

**Monitoring operators**
- removed assembly-operators-monitoring.adoc, included con-operators-prometheus-metrics.adoc procedure in assembly-operators.adoc

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/modules/ref-operator-topic.adoc [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+// assembly-using-the-topic-operator.adoc
+
+[id='ref-operator-topic-{context}']
+= Kafka topic resource
+
+The `KafkaTopic` resource is used to configure topics, including the number of partitions and replicas.
+
+The full schema for `KafkaTopic` is described in xref:type-KafkaTopic-reference[`KafkaTopic` schema reference].
+
+= Identifying a Kafka cluster for topic handling
```
Comment (scholzj) [inline documentation/modules/ref-operator-topic.adoc:12]: Should this be `==`?

File: documentation/modules/ref-operator-topic.adoc [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  name: topic-name-1
+  labels:
+    strimzi.io/cluster: my-cluster
+----
+
+The label is used by the Topic Operator to identify the `KafkaTopic` resource and create a new topic, and also in subsequent handling of the topic.
+
+If the label does not match the Kafka cluster, the Topic Operator cannot identify the `KafkaTopic` and the topic is not created.
+
+= Handling changes to topics
```
Comment (scholzj) [inline documentation/modules/ref-operator-topic.adoc:32]: Same as above, should this be `==` ... this seems to be on a lot of places ... not sure if this is right or wrong.

File: documentation/modules/ref-operator-topic.adoc (hunk 1 above)
Comment (PaulRMellor) [reply documentation/modules/ref-operator-topic.adoc:12]: I was looking at it from a TOC point of view. Maybe we can drop some of these sections to a lower level. I'll have another look.

File: documentation/modules/ref-operator-topic.adoc (hunk 2 above)
Comment (PaulRMellor) [reply documentation/modules/ref-operator-topic.adoc:32]: Yeah. I've moved this down a level. I originally left it like that to present these sections in the TOC, but I don't think its necessary.

**Comment by PaulRMellor** [comment]
@laidan6000 @ppatierno - could you take a look at this one?

File: documentation/modules/ref-operator-cluster.adoc [hunk 3]
```diff
@@ -120,3 +120,134 @@ _<cluster-name>_-kafka-0._<cluster-name>_-kafka-brokers._<namespace>_.svc._clust
...
+
+* The Kafka broker pods use a `ServiceAccount` called `_cluster-name_-kafka`
+  - When the rack feature is used, the `strimzi-_cluster-name_-kafka-init` `ClusterRoleBinding` is used to grant this `ServiceAccount` access to the nodes within the cluster via a `ClusterRole` called `strimzi-kafka-broker`
+  - When the rack feature is not used no binding is created
+* The ZooKeeper pods use a `ServiceAccount` called `_cluster-name_-zookeeper`
+* The Entity Operator pod uses a `ServiceAccount` called `_cluster-name_-entity-operator`
+    - The Topic Operator produces Kubernetes events with status information, so the `ServiceAccount` is bound to a `ClusterRole` called `strimzi-entity-operator` which grants this access via the `strimzi-entity-operator` `RoleBinding`
+* The pods for `KafkaConnect` and `KafkaConnectS2I` resources use a `ServiceAccount` called `_cluster-name_-cluster-connect`
+* The pods for `KafkaMirrorMaker` use a `ServiceAccount` called `_cluster-name_-mirror-maker`
+* The pods for `KafkaBridge` use a `ServiceAccount` called `_cluster-name_-bridge`
```
Comment (ppatierno) [inline documentation/modules/ref-operator-cluster.adoc:159]: There is also a service account for `KafkaMirrorMaker2` which is called `_cluster-name_-mirrormaker2`

Comment (PaulRMellor) [reply documentation/modules/ref-operator-cluster.adoc:159]: Added _The pods for `KafkaMirrorMaker2` use a `ServiceAccount` called `_cluster-name_-mirrormaker2`_

**Pull request merged**